commands/test_pull_requests_origin_check_future.sh

```diff
@@ -0,0 +1,9 @@
+# run commitchecker outside release container as it needs
+# access to git; also explicitly force godeps verification
+symbolic_ref="$( git symbolic-ref HEAD )"
+branch="${symbolic_ref##refs/heads/}"
+if [[ "${branch}" == "master" ]]; then
```

Comment:
@smarterclayton so before here we were using `${BRANCH}` which comes from the Jenkins environment.

I wasn't super happy with this -- so the API here would be, write the script in this file as if you were SSH-ed in. I'll escape `$` chars, etc. Don't try to make use of Jenkins envars, so for instance here I got the branch name from the git state.